app/views/companies/show.html.erb

```diff
@@ -3,6 +3,43 @@
   <%= link_to 'Stock', companies_path, class: "breadcrumb-item small text-decoration-none" %>
   <span class="breadcrumb-item active small"><%= @company.name %></span>
 </nav>
-<h2>Company name: <%= @company.name %></h2><br />
-<h4>Company symbol: <%= @company.isin %></h4><br />
-<h5 class="text-muted">Id in database: <%= @company.id %></h5>
+
+<% presenter = CompanyPresenter.new(@company) %>
```

Comment:
It would be better to do it in controller:

```ruby
  def index
    @companies = Company.all.map { |company| CompanyPresenter.new(company) }
  end
```